app/javascript/bundles/Presence/components/MovementsTable.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+class MovementsTable extends React.Component {
+
+  render() {
+    const {movements} = this.props;
+    return (
+      <div>
+        <table className="presence-table movements-table" cellpadding="0" cellspacing="0" style={{margin: '20px 0'}}>
```

Comment:
This style should be moved into CSS. There's also a `<Table>` component for wrapping tables in all the other components.